client/components/build/RightSidebar/sections/Theme.tsx

```diff
@@ -19,6 +19,9 @@ const Theme = () => {
   const { background, text, primary } = useAppSelector<ThemeType>((state) => get(state.resume, 'metadata.theme'));
 
   const handleChange = (property: string, color: string) => {
+    if (color[0] !== '#') {
+      color = `#${color}`;
```

Comment:
It'll be preferred to create a new temporary color variable and reassign that instead of modifying the prop directly.

Reply:
> Could this probably accept non hexadecimal values as well? Is it safe to assume that the value input it always a valid hex color?

The Current Behaviour is that if the input value is an INVALID hex, the color doesn't change
Tested with Invalid Hex Values like- ```12876( , ------, %*&*$^*,  #12345aa```

Reply:
Oh! I just read about it, It's bad practice to modify the prop directly. Thanks Amruth!